tests/integration/src/test/java/org/apache/pulsar/tests/integration/schema/SchemaTest.java

```diff
@@ -218,4 +220,43 @@ public void testAvroLogicalType() throws Exception {
 
     }
 
+    @Test
+    public void testAutoConsumeSchemaSubscribeFirst() throws Exception {
```

Comment:
I didn't see you are using AUTO_SCHEMA in this test.  So how does this test verify the change?